bwi_kr/Predicate fluent
---